exonum/src/runtime/blockchain_data.rs

```diff
@@ -40,14 +40,25 @@ impl<'a, T: RawAccess + AsReadonly> BlockchainData<'a, T> {
         }
     }
 
+    /// Returns unstructured readonly access to blockchain data.
+    ///
+    /// # Safety
+    ///
+    /// Trying to get access the executing service data can lead to panic.
```

Comment:
```suggestion
    /// Trying to access data of the executing service through the returned access
    /// can lead to a panic because of borrowing checks performed by the database.
```